File: Sem1/ProjetoIntegrador/exercicios/src/tarefaI/estudoDirigidoAlgoritimo/pg25_26Exercicio7/L01F.java
package tarefaI.estudoDirigidoAlgoritimo.pg25_26Exercicio7;
// f)    Ler dois valores (inteiros, reais ou caracteres) para as variáveis A e B,
// e efetuar a troca dos valores de forma que a variável A passe a possuir o valor da variável B,
// e a variável B passe a possuir o valor da variável A.
// Apresentar os valores trocados

import javax.swing.*;

public class L01F {
    static void main() {
        double firstValue;
        double secundValue;

        firstValue = Double.valueOf(JOptionPane.showInputDialog("Digite o valor de A: ").replace(",","."));
        secundValue = Double.valueOf(JOptionPane.showInputDialog("Digite o valor de B: ").replace(",","."));

        JOptionPane.showMessageDialog(null, "o valor de A: " + firstValue
                + "\no valor de B: " + secundValue);

        double trocador;
        trocador = firstValue;
        firstValue = secundValue;
        secundValue = trocador;

        JOptionPane.showMessageDialog(null, "Apos a troca:" +
                "\no valor de A: " + firstValue
                + "\no valor de B: " + secundValue);

    }

}

File: Sem1/ProjetoIntegrador/exerciciosPyton/L01 - pg25_26Exercicio7/L01F.py
#f) Ler dois valores (inteiros, reais ou caracteres) para as variáveis A e B, 
# e efetuar a troca dos valores de forma que a variável A passe a possuir o valor da variável B ,
# e a variável B passe a possuir o valor da variável A. Apresentar os valores trocados

valorA = str(input("Digite o valor da variavél A: "))
valorB = str(input("Digite o valor da variavél B: "))

intermediario = valorB
valorB = valorA
valorA = intermediario


print(f"\nO valor de A é: {valorA}\n")
print(f"\nOvalor de B é: {valorB}\n")         

